test/build/40_pull.bats

```diff
@@ -293,6 +293,12 @@ EOF
     # FIXME: 77 MiB unpacked, should find a smaller public image
     ch-image pull registry.access.redhat.com/ubi7-minimal:latest
 
+    # Microsoft Container Registry:
+    # https://hub.docker.com/publishers/microsoftowner
```

Comment:
This link is a bit confusing because it goes to Docker Hub but we're pulling from `mcr.microsoft.com`. Would the GitHub link I sent you work here?

Reply:
Sounds good. I will also remove the comment about Dockerhub being the official method to browse images since that information will be in the github link.